### PR TITLE
New scoutgame builders & scouts page layout

### PR DESCRIPTION
**Pull request by Devorein**
### WHY

[Card](https://app.charmverse.io/cvt-scout-game/builders-tab-main-app-8876382784058499)
[Card](https://app.charmverse.io/cvt-scout-game/new-scout-tab-main-app-3753646763789489)

### CODE REVIEW

File: apps/scoutgame/__e2e__/buyNft.spec.ts [hunk 1]
```diff
@@ -128,6 +128,6 @@ test.describe('Buy Nft', () => {
 
     // NFT buy button
     await expect(page.locator('data-test=purchase-button').first()).toBeDisabled();
-    expect(await page.locator('data-test=address-error')).toBeVisible();
+    expect(page.locator('data-test=address-error')).toBeVisible();
```
Comment (mattcasey) [inline apps/scoutgame/__e2e__/buyNft.spec.ts:131]: i think u still need await here

File: apps/scoutgame/app/(general)/page.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import dynamic from 'next/dynamic';
+
+const LandingPage = dynamic(() => import('components/home/LandingPage').then((mod) => mod.LandingPage), { ssr: false });
```
Comment (mattcasey) [inline apps/scoutgame/app/(general)/page.tsx:3]: why are we not loading anything on the server side?

**Review by mattcasey — APPROVED** [review]
approved but check my comments plz

**Comment by mattcasey** [comment]
when i click Search on Scouts, it puts an overlay only on half the page 
image

**Comment by mattcasey** [comment]
The right column should probably be fixed width and look more like this ratio:
image

File: apps/scoutgame/app/(general)/page.tsx (hunk 2 above)
Comment (Devorein) [reply apps/scoutgame/app/(general)/page.tsx:3]: fixed issue, it was client side rendered before due to using hooks but converted it to ssr version and its working fine

**Pull request merged**